### DISCUSSION TITLE
Accepted Member invitations stuck in Pending state

### DISCUSSION THREAD

**Discussion by eworkflow** [docmost/docmost#191] [Help / Questions]
For starters, Docmost concept is an excellent idea. I am testing Docmost using https://www.pikapods.com/ Docmost pod installation. Everything seems to be OK except that after sending Member invitations (to 2 different email clients:  Gmail and self-hosted one), I receive both emails, and I reply to accept each invitation (which is followed by being redirected to the Docmost portal). However, after reviewing Member invitation status in Docmost, I see that both Members are still (stuck?) in Pending state. Please advise.

**Comment by Philipinho** [comment]
The invite link should take you to the signup page. Did you complete the signup process?
